examples/bert/bert_ptq_cpu.json

```diff
@@ -49,9 +49,12 @@
         },
         "transformers_optimization": {
             "type": "OrtTransformersOptimization",
+            "disable_search": true,
             "config": {
-                "float16": false,
-                "only_onnxruntime": false
+                "model_type": "bert",
```

Comment:
BTW, you can remove model_type/num_heads/hidden_size/float16 in this case. Olive will extract and fill in these values from model_attributes.

Reply:
Removed.